configurations/lfhcal_withInlay.yml

```diff
@@ -0,0 +1,4 @@
+features:
+  hcal:
+    lfhcal_insertGeom:
+    forward_insert:
```

Comment:
This will cause overlaps since the forward_insert is the same as the insert.

Reply:
I don't see which "insert" you mean. "lfhcal_insertGeom.xml" is a file with the module definitions such that those which would overlap with the  insert are taken out of the geometry. Not the insert itself.  So there should not be a problem at all.

Reply:
Unclear name then.

Reply:
Per suggestion of @veprbl I now renamed the file to: `lfhcal_with_space_for_insert.xml`, hope this helps making things clearer.